diff --git a/openff/utilities/__init__.py b/openff/utilities/__init__.py
index ce7c288..503e368 100644
--- a/openff/utilities/__init__.py
+++ b/openff/utilities/__init__.py
@@ -15,7 +15,6 @@ from openff.utilities.utilities import (
 __all__ = (
     "MissingOptionalDependencyError",
     "get_data_file_path",
-    "get_versions",
     "has_executable",
     "has_package",
     "requires_oe_module",
